literature.bib

```diff
@@ -83,15 +98,65 @@ @inproceedings{STS:VaMoS20
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{PAP+:ICST19,
+  title = {{Uniform sampling of sat solutions for configurable systems: Are we there yet?}},
+  author = {Plazar, Quentin and Acher, Mathieu and Perrouin, Gilles and Devroey, Xavier and Cordy, Maxime},
+  booktitle = ICST,
+  pages = {240--251},
+  year = 2019,
+  doi = {10.1109/ICST.2019.00032},
+  sampling-tags = {evaluation, available tool, open-source tool, sampling efficiency, effectiveness, testing, non-functional properties, compares QuickSampler/UniGen},
+  organization = IEEE,
+  address = Piscataway 
+}
+
+@article{HNA+:ESE19,
+  title = {{Test them all, is it worth it? Assessing configuration sampling on the JHipster Web development stack}},
+  author = {Axel Halin, Alexandre Nuttinck and Mathieu Acher and Xavier Devroey and Gilles Perrouin and Benoit Baudry},
+  journal = {Empirical Software Engineering},
+  volume = {24},
+  number = {2},
+  pages = {674--717},
+  month = JUL,
+  sampling-tags = {evaluation, testing efficiency, effectiveness, testing, type checking, compares t-wise(SPLCAT)/one-disabled/one-enabled/most-enabled-disabled/dissimilarity(PLEDGE)/random, SPLC18},
+  year = 2019,
+  publisher = Springer
+}
+
+@inproceedings{KGS+:ICSE19,
+  title = {{Distance-based sampling of software configuration spaces}},
+  author = {Christian Kaltenecker and Alexander Grebhahn and Norbert Siegmund and Jianmei Guo and Sven Apel},
+  booktitle = ICSE,
+  pages = {1084--1094},
+  doi = {10.1109/ICSE.2019.00112},
+  isbn = {978-1-7281-0869-8},
+  sampling-tags = {name=Distance-Based Sampling, feature model, population-based search, semi-automatic search, evaluation, effectiveness, available tool, open-source tool, non-functional properties, compares distance-based/random/solver-based/coverage-based sampling strategies},
+  publisher = IEEE,
+  address = Piscataway,
+  year = 2019
+}
+
+@inproceedings{OGB:SPLC19,
+  title = {{t-wise Coverage by Uniform Sampling}},
+  author = {Jeho Oh and Paul Gazzillo and Don Batory},
+  booktitle = SPLC,
+  pages = {84--87},
+  tt-tags = {Product Sampling},
+  sampling-tags = {evaluation, sampling efficiency, testing efficiency, effectiveness, available tool, open-source tool, testing, submission to PTR+:SPLC19},
+  publisher = ACM,
+  month = SEP,
+  year = 2019
+}
+
 @techreport{OGB+:TR19,
-	author = {Jeho Oh and Paul Gazzillo and Don Batory and Marijn Heule and Maggie Myers},
 	title = {{Uniform Sampling from Kconfig Feature Models}},
+	author = {Jeho Oh and Paul Gazzillo and Don Batory and Marijn Heule and Maggie Myers},
```

Comment:
Why to make the order inconsistent with other orders?

Reply:
I don't know sometimes it just happens xD Changed all occurences